prep-javascript-variables/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <p>Not sure what I did wrong with Git Bash!  LOL!</p>
+
+  <script>
+    /* your code here */
+    var fullName = "Thomas Alexander";
+    console.log('Full Name:', fullName);
+
+    var numberOfPets = 0;
+    console.log('Number of Pets:', numberOfPets);
+
+    var likesVegetables = Boolean(10 >9)
+    console.log('Likes Vegetables:', likesVegetables);
+
+    var nothing = "null";
```

Comment:
You can leave out the quote marks here, otherwise, JavaScript interprets `"null"` as a string